gm4/plugins/output_release.py

```diff
@@ -0,0 +1,17 @@
+from beet import Context
+import os
+
+
+def beet_default(ctx: Context):
+	version = os.getenv("VERSION", "1.19")
+	base_path = f"release/{version}/{ctx.project_id}"
+	ctx.data.save(
+		path=f"{base_path}_{version}.zip",
+		overwrite=True,
+		zipped=True,
+	)
+
+	os.makedirs(base_path, exist_ok=True)
+	for file in ["README.md", "CREDITS.md", "pack.png"]:
+		if file in ctx.data.extra:
+			ctx.data.extra[file].dump(base_path, file)
```

Comment:
I would just have two plugins in the same `output` file:
```py
# output.py
def beet_default(ctx: Context):
  ...
  
def release(ctx: Context):
  ...

```

Reply:
Tbf, this is preference, but I like having less files for these smaller plugins since they group nicely together.
The same could work for the `create_*` and the `populate_*`, etc one as well.

In `beet`, you reference them the same `gm4.plugins.output.dev`!